src/main/services/outputProcessors/soot/Main.groovy

```diff
@@ -29,13 +34,15 @@ class Main {
             } else {
                 RunSootAnalysisOutputProcessor sootRunner = new RunSootAnalysisOutputProcessor();
 
-                if (appArguments.getAllanalysis()) {
-                    sootRunner.configureDetectionAlgorithmsTimeout(appArguments.getTimeout())
-                } else {
-                    sootRunner.setDetectionAlgorithms(configureDetectionAlgorithms(appArguments, sootWrapper))
-                }
+//                if (appArguments.getAllanalysis()) {
```

Comment:
Matheus, aqui precisamos fazer uma modificação, para quando o args for tipo: -orAnalysis, executar as análises que estão no atributo detectionAlgorithms da classe RunSootAnalysisOutputProcessor.groovy. Então ficaria três opções:

1. Run -a, -allAnalysis
2. Run -orAnalysis (Aqui executa somente as quatro análise e chama o método da linha 45: generateResults(outputPath)
3. Run specificy analys (-ioa, oa, dfp...)